man/fi_mr.3.md

```diff
@@ -546,6 +552,20 @@ region.  The domain authorization key will be used if the auth_key_size
 provided is 0.  This field is ignored unless the fabric is opened with API 
 version 1.5 or greater.
 
+## iface
+
+Indicates the interface used to allocate the memory region/device type:
```

Comment:
Indicates the software interfaces used by the application to allocate and manage the memory region.